### PR TITLE
store any public key related routines from go-btfs node to go-btfs-co…

### PR DESCRIPTION
**Pull request by laipogo**

### CODE REVIEW

**Comment by taiyangc** [comment]
I suggest testing the end-to-end flow first before merging, so we need to tag the current release, and update this PR https://github.com/TRON-US/go-btfs-escrow/pull/89

File: ledger/ledger.go [hunk 1]
```diff
@@ -128,3 +137,21 @@ func (c *Client) CloseChannel(ctx context.Context, signedChannelState *ledgerpb.
 		return err
 	})
 }
+
+func NewPublicKey(privK ic.PrivKey, pubK ic.PubKey) (*ledgerpb.SignedPublicKey, error) {
```
Comment (JustinGaoF) [inline ledger/ledger.go]: Suggest to change the name to make it more understandable.

Comment (taiyangc) [reply ledger/ledger.go]: NewSignedPublicKey?

Comment (laipogo) [reply ledger/ledger.go]: done the name change.

**Pull request merged**